### ISSUE TITLE
[Toolkit bug]: Giant obstructive light arrows

### ISSUE THREAD

**Issue by Frissj** [NVIDIAGameWorks/rtx-remix#596]
### Describe the bug

Pete tells me the arrows are not meant to be this big. Also they get in the way of the movement arrows so its extremely fiddley. They also loop around the entire capture. Please please can you make it like blender where the arrows are not overlapping anything. See image below.
![image](https://github.com/user-attachments/assets/6c7993fc-6cab-4287-8074-79358575cb59)

https://github.com/user-attachments/assets/0231aa19-8c86-4d40-999e-1a63daeffdcb

[kit_20240830_130220.log](https://github.com/user-attachments/files/16816829/kit_20240830_130220.log)

### How do you reproduce the bug?

Happens most of the time I am moving something.

### Version

2024.4.1

**Comment by nxkb** [comment]
Hi! Is this issue just happening with lights? 
It's possible you are seeing "intensity" indicators rather than arrow manipulators. Can you play with the light intensity value to see if it makes it more reasonably sized?
If the issue is just with light manipulators we might be able to give you some options like a scale control or just controlling their visibility.

**Comment by Frissj** [comment]
Yes they only happen with lights. I tried them and they do actually control the intensity. Is it possible to not make the arrow's overlap? And maybe a toggle in settings to hide it? Or make it shrink back to normal size everytime after I release the mouse?

**Comment by nxkb** [comment]
Thanks for filing @Frissj 
I've created an internal ticket for tracking: REMIX-3553

**Comment by nxkb** [comment]
Hi @Frissj . There is now a new option under "Show By Type" > "Lights" > "Intensity Controls" that controls the arrows. They are now off by default to keep your viewport clean unless your are working with intensity.

Try it on the latest builds or wait to test it in the next release (1.5). Closing this but feel free to re-open if your issue persists.
